.circleci/config.yml

```diff
@@ -60,6 +60,20 @@ aliases:
     attach_workspace:
       at: ~/datadog
 
+  - &STEP_APPEND_BUILD_ID
+    run:
+      name: Append build id to version number
+      command: |
+        githash="${CIRCLE_SHA1?}"
+        if [[ "$CIRCLE_BRANCH" =~ "ddtrace-" ]] ; then
+          echo "Release branch detected; not adding git sha1 to version number."
+        else
+          sed -E -i "s/const\s+VERSION\s*=\s*'(.*)'/const VERSION = '\1+$githash'/g" "src/DDTrace/Tracer.php"
```

Comment:
Considering this modifies sources, can we easily make it part of the build process ? Like a different makefile target ?

Reply:
we talked about potential improvements to unify the version management across languages (rust / php / C) which would involve having a make target to propagate version numbers.
Accepting PR for now as this won't change the result of the build.